indico/modules/events/registration/blueprint.py

```diff
@@ -77,6 +77,12 @@
                  'hide_registration', reglists.RHRegistrationHide, methods=('POST',))
 _bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/<int:registration_id>/withdraw',
                  'manage_withdraw_registration', reglists.RHRegistrationManageWithdraw, methods=('POST',))
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/<int:registration_id>/remove-allow-modification',
```

Comment:
Maybe rename the url to something that is a bit more concise.

```suggestion
_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/<int:registration_id>/deny-modification',
```

Reply:
I agree, I wanted to change to this but I figured `close` would make more sense than `deny` since the manager is not reviewing a request.